src/lib/functions/form-submissions-handler.js

```diff
@@ -38,7 +39,7 @@ const createFormSubmissionHandler = function ({ functionsRegistry, siteUrl }) {
         encoding: ct.parameters.charset,
       })
 
-      fields = getObjectFromParams(new URLSearchParams(bodyData.toString()))
+      fields = fromEntries([...new URLSearchParams(bodyData.toString())])
```

Comment:
```suggestion
      fields = fromEntries(new URLSearchParams(bodyData.toString()))
```

[sand] Tiny simplification since `URLSearchParams()` is iterable and `Object.fromEntries()` can use iterables as argument.